js/src/block-editor/controls/color.js

```diff
@@ -76,11 +33,33 @@ const GcbColorControl = ( props ) => {
 				value={ value }
 				onChange={ onChange }
 			/>
-			<GcbColorPopover
-				isVisible={ false }
-				color={ value }
-				onUpdate={ onChange }
-			/>
+			<BaseControl
+				className="genesis-custom-blocks-color-popover"
+				id={ __( 'Color control picker', 'genesis-custom-blocks' ) }
+			>
+				<ColorIndicator
+					colorValue={ value }
+					onMouseDown={ ( event ) => {
+						event.preventDefault(); // Prevent the popover blur.
+					} }
+					onClick={ () => setIsVisible( true ) }
+				/>
+			</BaseControl>
+			{ isVisible
+				? (
+					<Popover
+						onClick={ ( event ) => event.stopPropagation() }
+						onClose={ () => setIsVisible( false ) }
+					>
+						<ColorPicker
```

Comment:
Maybe not for this PR, but could we use the default theme colours here by pulling them via:
`wp.data.select( 'core/block-editor' ).getSettings().colors`?

Reply:
Oh interesting, good point

Reply:
Feel free to resolve, as I guess it's out of scope.

Reply:
Sure, thanks